src/app/layout.js

```diff
@@ -0,0 +1,28 @@
+import localFont from "next/font/local";
+import "./globals.css";
+
+const geistSans = localFont({
```

Comment:
Check if we can remove this file and move this to common page level

Reply:
Done